include/clang/AST/ASTContext.h

```diff
@@ -2486,12 +2488,16 @@ class ASTContext : public RefCountedBase<ASTContext> {
   /// at least as much checking as the other type T2.  Return true if it does
   /// or false if it does not or the types differ in some other way than
   /// checkedness.
+  /// Note:: Checked C - In bounds safe interface scopes, this function
+  /// returns true if T2 is a TypeVariableType and T1 is a pointer to void type
   bool isAtLeastAsCheckedAs(QualType T1, QualType T2) const;
 
   /// \brief Determine whether a pointer, array, or function type T1
   /// is the same as the other pointer, array, or function type T2 if
   /// checkedness is ignored.  Return true if does or false if the types
   /// differ in some other way than checkedness.
+  /// Note:: Checked C - In bounds safe interface scopes, this function
```

Comment:
You don't need to prefix this with "Checked C", as this function is specific to the Checked C implementation.